docs/2021-05-27-debug-feature-tests.md

```diff
@@ -0,0 +1,14 @@
+# Debug feature tests
+
+Add a `binding.pry` into the context of a test that you want to investigate.
+
+Copy the name of the test (relative path) of the test in question, run `rspect` in your terminal with the `CHROME` flag.
+
+For example, I want to run the `clients_searching_sorting_and_filtering_spec` spec:
+
+```bash
+# In the root of the vita-min project
+CHROME=y rspec ./spec/features/hub/clients_searching_sorting_and_filtering_spec.rb
+```
+
+This will open a chrome browser to the context that you added your binding. Now you can execute commands in terminal and see them work in the browser.
```

Comment:
Since you made a video and uploaded it to the GitHub pull request, I believe you can link to https://user-images.githubusercontent.com/9101728/119901780-fa4fa080-bf0b-11eb-86d4-a29cb77a8b9f.mp4 (the URL of the video) from the docs. I encourage you to do that.